iam-login-service/src/main/java/it/infn/mw/iam/api/client/management/ClientManagementAPIController.java

```diff
@@ -140,6 +141,13 @@ public RegisteredClientDTO updateClient(@PathVariable String clientId,
     return managementService.updateClient(clientId, client);
   }
 
+  @PatchMapping("/{clientId}/status")
```

Comment:
Maybe a PUT?

Reply:
I have used PATCH as it updates a resource without sending the entire body in the request.